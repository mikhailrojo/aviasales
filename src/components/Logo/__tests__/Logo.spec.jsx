import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {Logo} from '../index';

describe('Snapshot', () => {
	it('render Logo', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(<Logo />);
		expect(component).toMatchSnapshot();
	});
});
