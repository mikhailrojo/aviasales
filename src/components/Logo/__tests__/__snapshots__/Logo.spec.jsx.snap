// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render Logo 1`] = `<styled.div />`;
